ports/spix/portfile.cmake

```
@@ -0,0 +1,27 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO faaxm/spix
+    REF v0.2
+    SHA512 14eb742b7861d510466341f90f8d5b9d519aeaf27a032a8be8ab15743c7dd20d0584aa1f815a82dd54e73cb747612975f4a52db23c57390e9b5cd4a102a789c6
+    HEAD_REF master
+    PATCHES
+        export-header.patch
+)
+
+string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "dynamic" ANYRPC_LIB_BUILD_SHARED)
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+    OPTIONS
+        -DSPIX_BUILD_EXAMPLES=OFF
+        -DBUILD_TESTS=OFF
```

```suggestion
        -DSPIX_BUILD_TESTS=OFF
```
https://github.com/faaxm/spix/blob/12198f9baa61c811dcc0f5d9b7fe8f4acc78f28c/CMakeLists.txt#L16

---

good catch, thanks